packages/mobile/locales/es-419/nuxVerification2.json

```
@@ -12,60 +12,60 @@
-    "card4": "Requerimos tres mensajes de confirmación para mantener su cuenta más segura."
+    "card1": "Para asegurarnos de que tu número sea realmente tuyo, te enviaremos tres mensajes de texto.",
+    "card2": "Una vez que se confirma tu número, puede enviar y recibir fondos fácilmente a tu número de teléfono.",
+    "card3": "Confirmar también le permite ver cuáles de tus amigos están en {{appName}}.",
+    "card4": "Requerimos tres mensajes de confirmación para mantener tu cuenta más segura."
   },
   "skipModal": {
     "header": "¿Omitir la verificación?",
-    "body1": "La verificación permite que otras personas le envíen valores a su número telefónico.",
+    "body1": "La verificación permite que otras personas le envíen valores a tu número telefónico.",
     "body2": "Sin la verificación, igual podrá recibir pagos, pero solo mediante las direcciones de cuenta o códigos QR."
   },
   "failModal": {
     "header": "Error de verificación",
-    "body1": "Se produjo un problema al verificar su número de teléfono.",
+    "body1": "Se produjo un problema al verificar tu número de teléfono.",
     "body2": "Puede saltar la verificación y volver a intentarlo más tarde.",
-    "body1InsufficientBalance": "Su saldo actual no es lo suficientemente alto como para completar el proceso de verificación.",
-    "body2InsufficientBalance": "Intente nuevamente cuando haya agregado fondos a su billetera.",
-    "body1SaltQuotaExceeded": "Su cuota de búsqueda de número de teléfono está agotada",
+    "body1InsufficientBalance": "Tu saldo actual no es lo suficientemente alto para completar el proceso de verificación.",
+    "body2InsufficientBalance": "Intenta nuevamente cuando hayas agregado fondos a tu app.",
+    "body1SaltQuotaExceeded": "Tu cuota de búsqueda de número de teléfono está agotada",
     "body2SaltQuotaExceeded": "Intente nuevamente cuando haya comprado más búsquedas",
     "enterCodesBody": "Si ya recibió tres códigos por SMS, aún puede ingresarlos.",
     "enterCodesButton": "Ingresar códigos"
   },
   "retryWithFornoModal": {
     "header": "¿Volver a intentar la verificación con Ahorro de Datos?",
-    "body1": "Se ha producido un problema al verificar su número de teléfono. Recomendamos volver a intentarlo con el modo Ahorro de Datos habilitado",
+    "body1": "Se ha producido un problema al verificar tu número de teléfono. Recomendamos volver a intentarlo con el modo Ahorro de Datos habilitado",
     "body2": "Ahorro de Datos le permite comunicarse en la red de Celo de manera más consistente utilizando una conexión confiable",
     "retryButton": "Reintentar con Ahorro de Datos"
   },
   "congratsVerified": "¡Felicitaciones se ha verificado tu usuario!",
   "notification": {
     "title": "Confirma el Teléfono",
-    "body": "Reciba pagos directamente a su número de teléfono",
+    "body": "Reciba pagos directamente a tu número de teléfono",
```

```suggestion
    "body": "Recibe pagos directamente a tu número de teléfono",
```

---

> ```suggestion
> 
>     "body": "Recibe pagos directamente a tu número de teléfono",
> 
> ```

No tengo acceso a ese número desde hace dos años